### PR TITLE
15 update sql style guide with sqlfluff rulesnotes

### LINKED ISSUE #15: Update SQL Style guide with SQLFLUFF rules/notes
Include sqlfluff rule # and if it is sqlfluff fix compatible

### PR DESCRIPTION
**Pull request by jtomkins**
## Description & motivation:
Enhancements to UCSC Style guide and SQLFluff upgrade included more rules we are interested in linting and fixing. 
Changes to Sql Style Guide to include the corresponding sqlfluff rules
New SQLFluff documentation to include sql style guide rules 

## To-do before merge:
n/a

## Screenshots:
n/a

## References or links:
issue #4, #15, and https://github.com/cdp-ucsc/dbt-pilot/issues/66, https://github.com/cdp-ucsc/dbt-pilot/issues/102

## Validation of models:
n/a

## Checklist:
- [x] I filled out my PR.
- [x] I assigned a reviewer(s) to my PR.
- [x] My pull request represents one logical piece of work.
- [x] My commits are related to the pull request and look clean.
- [ ] My SQL follows the [UCSC Style Guide (wip-draft)](https://github.com/cdp-ucsc/dbt-pilot/wiki/Style-Guide-Draft).
- [ ] I have materialized my models appropriately.
- [ ] I have added appropriate tests and documentation to any new models.
- [ ] I have updated the README file.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204585020717576

### CODE REVIEW

**Review by smikawa-ucsc — APPROVED** [review]
Looks great!

File: cdp-docs/docs/project_resources/sql_style_guide.md [hunk 1]
```diff
@@ -341,7 +371,9 @@ with paying_customers as (
 select ...
 from paying_customers_per_month
 ```
-
+| **SQLFluff Rule Code** | **SQLFluff Fix compatible** |
+|:---|:---|
+| [LT07](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT07), <br /> [LT08](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT08) | Yes |
```
Comment (insuhpak) [inline cdp-docs/docs/project_resources/sql_style_guide.md]: Is the `<br />` intentional? Wondering bc it doesn't match the format of the other listed rule codes.

Comment (jtomkins) [reply cdp-docs/docs/project_resources/sql_style_guide.md]: Hi Insuh -- It was intentional when I thought I was keeping one rule code per line, I missed this one, in the cleanup.  Thank you!

**Comment by insuhpak** [comment]
Hi Jen looks great!! Love the inline reference to the sqlfluff rule code it is so convenient.

I was able to see the changes really clearly by building a local site off of the PR branch.
FYI, something I noticed is that the current version of the website looks like this (notice the left sidebar)
image

And when I build this PR branch locally the website looks like this
image

So I actually merged main into this PR branch (just locally for me) to see what the site would actually look like with this PR's changes
image

Doesn't impact this PR but wanted to mention it for others.

**Review by insuhpak — APPROVED** [review]

**Review by magants — APPROVED** [review]
Nice work Jen!

**Review by tgraham-ucsc — APPROVED** [review]
Nice work Jen - looks great!

**Pull request merged**